config/initializers/doorkeeper.rb

```diff
@@ -0,0 +1,106 @@
+Doorkeeper.configure do
```

Comment:
most of `config/initializers/doorkeeper.rb` is generated automatically by running
`rails generate doorkeeper:install`